src/components/tag-input/tag-input.component.less

```diff
@@ -109,6 +109,11 @@ ux-tag-input {
 
     &.disabled {
         background-color: @tag-input-disabled-bg;
+
+        .ux-tag-icons .ux-tag-icon, .ux-tag-icons .ux-select-icon {
```

Comment:
We use Less and Sass to eliminate repetition, so we can write the above as:

````less
.ux-tag-icons {
    .ux-tag-icon,
    .ux-select-icon {
````

(We also tend to split comma-separated selectors with a new line as well, just to emphasize the different cases when reading the code.)

Reply:
updated